tdrs-backend/tdpservice/users/test/test_auth_check.py

```diff
@@ -6,10 +6,11 @@
 
 
 @pytest.mark.django_db
-def test_auth_check_endpoint_with_no_user(api_client, user):
-    """If there is no user auth_check should return FORBIDDEN."""
+def test_auth_check_endpoint_with_no_user(api_client):
+    """If there is no user auth_check should return 200 with unauthorized message."""
     response = api_client.get(reverse("authorization-check"))
-    assert response.status_code == status.HTTP_403_FORBIDDEN
```

Comment:
@carltonsmith Could you say more about this change, and how this relates to the story and ACs? Changes to authorization results always catch my eye. 

Is this test covering an authorization case that should be documented in `authorization_check.md`?

Reply:
@alexsoble This was updated because of an adjustment to permissions. Previously it was giving a 403 on unauthorized users, but this endpoint was actually designed to be open and check to see itself if the user was authorized. So when the permissions were corrected, the authorization stopped returning a 403, but a 200 with a message that says the user is unauthorized. The tests now reflect both of those things.

The way it's related to the current ACs is that the permission update was made in support of the endpoint we created with this ticket.